app/Http/Controllers/ProfilesController.php

```diff
@@ -74,12 +79,39 @@ public function edit($id)
      * Update the specified resource in storage.
      *
      * @param  \Illuminate\Http\Request  $request
-     * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, $id)
+    public function update(Request $request)
     {
-        //
+        $input = array_except(Input::all(), '_method');
+
+        // Should be in a model or form?
+        $validator = Validator::make($request->all(), [
```

Comment:
Thanks!

In this case I think its ok, since we are doing a very specific check in a form that wont be reused.